qphix_interface.cpp

```diff
@@ -78,9 +86,17 @@ extern "C" {
 #include <qphix/wilson.h>
 #include <qphix/twisted_mass.h>
 
+#include <vector>
+
+// XXX Can we please not do that? tmLQCD and QPhiX are libraries of the same domain and define the
+// same stuff with slightly different names. One has to be an expert in both libraries to know where
+// a symbol comes from. Martin would prefer to have `QPhiX::` in front of the QPhiX types and
+// functions.
 using namespace std;
 using namespace QPhiX;
 
+// XXX The following block of code also occurs many times in QPhiX. It seems appropriate to put that
+// into the QPhiX library.
```

Comment:
no, I would like the interface to check if the headers define the most important preprocessor constant